code-studio/src/js/levelbuilder.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file Main entry point for scripts used on all level editing pages.
+ */
+'use strict';
+var _ = require('lodash');
+
+window.levelbuilder = window.levelbuilder || {};
+_.extend(window.levelbuilder, {
+  initializeCodeMirror: require('./initializeCodeMirror')
+});
+
+// TODO: Remove when global `CodeMirror` is no longer required.
+window.CodeMirror = require('codemirror');
+// TODO: Remove when global `marked` is no longer required.
+window.marked = require('marked');
```

Comment:
Should we track these somewhere? For marked at least I don't think we get to that point until apps is able to require from code-studio.